### PR TITLE
Minor edits for clean compile

### PR DESCRIPTION
**Pull request by bobgates**
I've also put a bad PR in for the same things against the release branch. Still learning how to do PRs properly.

I've added dependency to image_transport and included a header that was required for a clean compile.

### CODE REVIEW

**Comment by Michael-Equi** [comment]
@bobgates Could you go ahead and merge with the new code that I just pushed and get your PR passing the travis build?

File: src/t260.cpp [hunk 1]
```diff
@@ -1,6 +1,8 @@
 /*
 MIT License
 
+
+=======
```
Comment (Michael-Equi) [inline src/t260.cpp]: Please remove the merge artifacts

**Review by Michael-Equi — CHANGES_REQUESTED** [review]
Looks good just go ahead and fix that merge artifact

**Review by bobgates — COMMENTED** [review]
Merge artefact removed

**Review by Michael-Equi — APPROVED** [review]
Looks good. Thank you for your contribution.

**Pull request merged**